app/javascript/lib/visualizations/modules/bubble_legend.js

```diff
@@ -64,7 +64,7 @@ export class VisBubbleLegend {
       .enter()
       .append('stop')
       .attr('stop-color', d => d)
-      .attr('offset', d => `scale(${d})%`);
+      .attr('offset', d => { return scale(d) + '%'; });
```

Comment:
Thanks for the fix. A small suggestion for the `arrow function` 

```suggestion
      .attr('offset', d => `${scale(d)}%`);
```